Jenkinsfile

```
@@ -125,7 +125,8 @@ for (int i = 0; i < splits.size(); i++) {
             def image = skipImageBuild ? docker.image('jenkins/ath') : docker.build('jenkins/ath', '--build-arg uid="$(id -u)" --build-arg gid="$(id -g)" ./src/main/resources/ath-container/')
             sh 'mkdir -p target/ath-reports && chmod a+rwx target/ath-reports'
             def cwd = pwd()
-            image.inside("-v /var/run/docker.sock:/var/run/docker.sock -v '${cwd}/target/ath-reports:/reports:rw' --shm-size 2g") {
+            def dockergid = sh label: 'get docker group', returnStdout: true, script: 'getent group docker | cut -d: -f3'
+            image.inside("--group-add ${dockergid} -v /var/run/docker.sock:/var/run/docker.sock -v '${cwd}/target/ath-reports:/reports:rw' --shm-size 2g") {
```

use the supported https://docs.docker.com/engine/containers/run/#additional-groups

---

```suggestion
            image.inside("--group-add $(getent group docker | cut -d: -f3) -v /var/run/docker.sock:/var/run/docker.sock -v '${cwd}/target/ath-reports:/reports:rw' --shm-size 2g") {
```

Would it avoid to execute the shell step before (to avoid a back and forth) ? I remember that `$(id -u)` was used on other cases

---

the would appear to me to be very brittle and relies on the image.inside implementation that would currently go via a shell to interpret this.   (which is why I moved it outside).